src/components/BulkManagementTab/ResultsSummary.jsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable react/sort-comp, react/button-has-type */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Hyperlink, Icon } from '@edx/paragon';
+import { Download } from '@edx/paragon/icons';
+
+import { bulkGradesUrlByCourseAndRow } from 'data/constants/api';
+
+/**
+ * <ResultsSummary {...{ courseId, rowId, text }} />
+ * displays a result summary cell for a single bulk management upgrade history entry.
+ * @param {string} courseId - course identifier
+ * @param {number} rowId - row/error identifier
+ * @param {string} text - summary string
+ */
+const ResultsSummary = ({
+  courseId,
+  rowId,
+  text,
+}) => (
+  <Hyperlink
+    href={bulkGradesUrlByCourseAndRow(courseId, rowId)}
+    destination="www.edx.org"
```

Comment:
what?

Reply:
in order to user Hyperlink without complaints, we must have a destination, even if not used... is weird but has no functionality impact